src/components/AddBook.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { nanoid } from '@reduxjs/toolkit';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
```

Comment:
- [ ] Following the best practices, you are not meant to leave unused code as a comment in your codebase. it makes your code look a bit unprofessional, I'll suggest you remove these. :smile: :small_airplane: